.github/workflows/release-demo.yaml

```diff
@@ -33,7 +33,7 @@ jobs:
 
       - name: Setup
         if: steps.check.outputs.changed == 'true'
-        uses: swisspost/design-system/.github/actions/setup-pnpm@main
+        uses: swisspost/design-system/.github/actions/setup-pnpm@release/angular-demo-archive
 
       # Install changesets locally
       - name: Install demo app & dependencies
```

Comment:
On step "name: Publish demo app to netlify" perhaps you can already duplicate the environment variable for the site ID and rename with something like "NETLIFY_ARCHIVE_SITE_ID" so it's clearer where it is deployed?